source/index.js

```diff
@@ -95,7 +95,6 @@ module.exports = async (input = 'patch', options) => {
 	const tasks = new Listr([
 		{
 			title: 'Prerequisite check',
-			enabled: () => runPublish,
```

Comment:
Why was this added in the first place?

Reply:
`git blame` is your friend for questions like this. https://github.com/sindresorhus/np/commit/24fa0cd26b296f9510e971d599d558a6c6748125#diff-168726dbe96b3ce427e7fedce31bb0bcR51

Why are you removing it? It was added for a good reason.